engines/sword25/metaengine.cpp

```diff
@@ -95,6 +101,86 @@ SaveStateList Sword25MetaEngine::listSaves(const char *target) const {
 	return saveList;
 }
 
+Common::KeymapArray Sword25MetaEngine::initKeymaps(const char *target) const {
+	using namespace Common;
+	using namespace Sword25;
+
+	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "sword25-default", _("Default keymappings"));
+	Keymap *debugKeyMap = new Keymap(Keymap::kKeymapTypeGame, "debug", _("Debug keymappings"));
+
+	Common::Action *act;
+
+	act = new Action(kStandardActionLeftClick, _("Move / Interact / Skip dialog"));
+	act->setLeftClickEvent();
+	act->addDefaultInputMapping("MOUSE_LEFT");
+	act->addDefaultInputMapping("JOY_A");
+	engineKeyMap->addAction(act);
+
+	act = new Action("SKIPDLG", _("Examine / Skip dialog"));
+	act->setRightClickEvent();
+	act->addDefaultInputMapping("MOUSE_RIGHT");
+	act->addDefaultInputMapping("JOY_B");
+	engineKeyMap->addAction(act);
+
+	act = new Action(kStandardActionPause, _("Pause"));
+	act->setKeyEvent(KeyState(KEYCODE_p, 'p'));
+	act->addDefaultInputMapping("p");
+	act->addDefaultInputMapping("JOY_RIGHT_SHOULDER");
+	engineKeyMap->addAction(act);
+
+	act = new Action("REVEALHOTSPOTS", _("Reveal all interactive hotspots (hold)"));
```

Comment:
Would it be possible to expand on "hold" for clarity? Presumably it means that the key/button needs to be held down in order to take effect?

Reply:
```suggestion
	act = new Action("REVEALHOTSPOTS", _("Reveal all interactive hotspots (hold the key)"));
```